src/components/SingleArticle.jsx

```diff
@@ -1,32 +1,47 @@
 import { useParams } from "react-router-dom";
 import { useEffect, useState } from "react";
 import * as api from "../api";
+import CommentsCard from "./CommentsCard";
 
 const SingleArticle = () => {
     const { article_id } = useParams();
     const [article, setArticle] = useState({});
-    const [isLoading, setIsLoading] = useState(true)
+    const [comments, setComments] = useState([]); 
+    const [isLoading, setIsLoading] = useState(true);
 
     useEffect(() => {
         api.fetchSingleArticle(article_id).then((article) => {
             setIsLoading(false);
             setArticle(article);
         })
         .catch((err) => {
-            console.log(err)
-        })
-    }, [])
+                console.log(err)
+            });
+        
+        api.fetchComments(article_id).then(comments => {
+                setIsLoading(false)
+                setComments(comments)
```

Comment:
You're doing this here to fetch the comments but you have a whole _CommentsList_ component set up to do the same thing that doesn't seem to be getting used